tests/network/curl_server.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2015-2018 SUSE Linux GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Server configurations specifit to test curl .
+# Maintainer: Marcelo Martins <mmartins@@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use mm_network 'setup_static_mm_network';
+use mm_tests;
+use testapi;
+use lockapi qw(mutex_create mutex_wait);
+use mmapi;
+
+
+sub run {
+    my $self = shift;
+    select_console 'root-console';
+    #preparing files will be use by client side.
+    script_run(' echo "Hello World!!" > /srv/www/htdocs/get');
+    script_run(' echo "Hello World!!" > /srv/ftp/file_test');
```

Comment:
Why `script_run` it is silently failing http://10.161.228.111/tests/1191#step/curl_server/4

Reply:
`assert_script_run` is better, script run is used in conditions or when command can fail but it is not important

Reply:
ok, done